framework/src/controller/bus.js

```diff
@@ -133,6 +134,37 @@ class Bus extends EventEmitter2 {
 		};
 	}
 
+	/**
+	 * Unregister channel from bus.
+	 *
+	 * @async
+	 * @param {string} moduleAlias - Alias for module used during registration
+	 *
+	 * @throws {Error} If channel not registered.
+	 */
+	async unregisterChannel(moduleAlias) {
+		Object.keys(this.events).forEach(eventName => {
```

Comment:
maybe it's a bit dangerous to dynamically delete within `forEach`?
prefer to get the `eventName` which needs to be deleted first and then delete?
same for actions

Reply:
Any particular reason for that? Why is it dangerous?

Reply:
As discussed, i think it's fine